code/modules/power/gravitygenerator.dm

```diff
@@ -453,6 +458,10 @@ GLOBAL_LIST_EMPTY(gravity_generators)
 	for(var/obj/machinery/gravity_generator/part as anything in generator_parts)
 		SET_PLANE(part, PLANE_TO_TRUE(part.plane), new_turf)
 
+/obj/machinery/gravity_generator/main/proc/get_radioactive_nebula_shielding()
+	SIGNAL_HANDLER
```

Comment:
Callbacks don't need signal handlers
Also may doc these that they're used as callbacks